docs/resources/cce_addon.md

```diff
@@ -6,7 +6,7 @@ subcategory: "Cloud Container Engine (CCE)"
 
 Provides a CCE add-on resource within HuaweiCloud.
 
-## Example Usage
+## Basic Example
```

Comment:
`Example Usage` is a fixed section in our docs, so it's better to add a sub-title of `Basic Example`

```suggestion
## Example Usage

### Basic Example
```

Reply:
done